### PR TITLE
🦀 MASTER IS GONE 🦀

### PR DESCRIPTION
**Pull request by Doges**

### CODE REVIEW

**Comment by nitrog0d** [comment]
eah

File: src/Switchblade.js [hunk 1]
```diff
@@ -20,8 +20,7 @@ module.exports = class Switchblade extends Client {
    * @param {string} [token] - Token of the account to log in with
    * @returns {Promise<string>} Token of the account used
    */
-  login (token) {
-    token = token || process.env.DISCORD_TOKEN
+  login (token = process.env.DISCORD_TOKEN) {
```
Comment (MrJacz) [inline src/Switchblade.js]: discord.js already looks for DISCORD_TOKEN

File: src/Switchblade.js [hunk 2]
```diff
@@ -60,8 +59,8 @@ module.exports = class Switchblade extends Client {
   }
 
   async initializeLoaders () {
-    for (let Loader of Object.values(Loaders)) {
-      const loader = new Loader(this)
+    for (let name in Loaders) {
```
Comment (MrJacz) [inline src/Switchblade.js]:
```suggestion
    for (const name of Object.values(Loaders)) {
```

File: package.json [hunk 3]
```diff
@@ -32,6 +32,7 @@
   },
   "homepage": "https://github.com/SwitchbladeBot/switchblade#readme",
   "dependencies": {
+    "@discordjs/uws": "^11.149.1",
```
Comment (MrJacz) [inline package.json]: Do not use uws, it does not actually help and is broken.

**Pull request merged**